library.wdl

```diff
@@ -42,7 +43,31 @@ workflow Library {
             reference = chipSeqInput.reference
     }
 
+    call samtools.View as filterBam {
+    ## Remove reads unmapped, mate unmapped, not primary alignment, reads failing platform, duplicates (using
+    ## excludeFilter)
+    ## Remove multi-mapped reads (with MAPQthreshlod)
```

Comment:
Is it necessary to remove these reads? They may be interesting for the QC metrics.

And just as a style thing, could you indent these comments? Everything within `{}` should be indented.